src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -597,51 +602,49 @@ export class BrowsingContextProcessor {
   ): Network.AddInterceptResult {
     // TODO: Implement.
     return {
-      result: {
-        intercept: '',
-      },
+      intercept: '',
```

Comment:
@thiagowfx Next time we do this we should throw error that the command is not supported.